### PR TITLE
Contrast for pagination focus outlines

### LINKED ISSUE #152: Improve pagination focus outlines contrast in schedule
On the schedule page, the pagination buttons have a light yellow focus outline with a contrast of approximately 1.5:1. 

Screen Shot 2022-03-15 at 4 24 09 PM

One possible fix for this would be to use the same styling on the "AM/PM" navigation that is on the workshops page - the contrast on this is better (though still appears to be around 3.3:1).

### PR DESCRIPTION
**Pull request by kaitlinnewson**
Improve the contrast for both the schedule pagination and workshop time pagination outlines. Also adds the schedule pagination to the workshop page for easier transition between the schedule pages. 

To test, navigate through the workshop and schedule pages with keyboard tabbing and test the contrast between the focus outline and the background of that page and button colours.

Resolves #152.

### CODE REVIEW

**Comment by phette23** [comment]
Sorry that commit was just the wrong number, I force-pushed to overwrite it so you'll see it's orphaned now.

Is there a way to test numerical contrast ratios? The contrast looks great but I wasn't sure if you wanted to test with a particular tool. The accessibility check in Chrome dev tools doesn't check focus/hover I think.

**Comment by kaitlinnewson** [comment]
> Is there a way to test numerical contrast ratios? The contrast looks great but I wasn't sure if you wanted to test with a particular tool. The accessibility check in Chrome dev tools doesn't check focus/hover I think.

I used this tool to check the contrast individually: https://www.tpgi.com/color-contrast-checker/

File: assets/_scss/_presentations-workshops.scss [hunk 1]
```diff
@@ -91,6 +91,21 @@
...
+  & button {
+    border: 1px solid $gray-light;
+
+    &:hover,
+    &:active {
+      background-color: $accent;
+    }
+
+    &:focus {
+      background-color: $accent;
```
Comment (phette23) [inline assets/_scss/_presentations-workshops.scss:104]: The pagination menu and the workshop sorting one function differently in terms of their background colors and selection state. Pagination: selected element has accent bg, focused element has border, other elements have neither. Workshop sorting: selected element has a light gray bg, focused element has border _and accent bg_, other elements have neither.

Would it makes sense to have both operate the same? I like the way pagination works if given a choice between the two, I think having the accent color bg is much clearer than a gray bg that can be confused with some sort of "inactive" state.

**Review by phette23 — CHANGES_REQUESTED** [review]
See my comments: I think we should make pagination and workshop sorting work the same, and I wonder if `day: day0` makes more sense for the workshops index than `day: day5`.

Also I noticed that the middle "AM" sort-workshop-time button does not have a right border when focused:
Screen Shot 2022-03-24 at 11 24 18 AM

File: workshops/index.html [hunk 2]
```diff
@@ -1,6 +1,7 @@
 ---
 layout: default
 title: Pre-Conference Workshops
+day: day5
```
[the remark this answers is not in the document]
Comment (kaitlinnewson) [reply workshops/index.html:4]: Yes, this is a holdover from last year. What do you think about using `day0`? Or I could change it to `day1` and adjust the other pages.

[the remark this answers is not in the document]
Comment (phette23) [reply workshops/index.html:4]: I was thinking `day0` makes the most sense, I like how it kind of sets the pre-conferences apart from the main conference days and makes the references elsewhere logical (first day is day1, etc.). Thanks!

**Comment by kaitlinnewson** [comment]
@phette23 this is ready for a second review now

**Review by phette23 — APPROVED** [review]
Awesome! This looks so much better.

**Pull request merged**